packages/amplify-data-construct/.jsii

```diff
@@ -10,11 +10,9 @@
     "@aws-amplify/backend-output-schemas": "^0.4.0",
     "@aws-amplify/backend-output-storage": "^0.2.2",
     "@aws-amplify/graphql-auth-transformer": "4.0.0",
-    "@aws-amplify/graphql-conversation-transformer": "0.1.0",
```

Comment:
This is not expected right? You might need to refresh these files.

Reply:
`yarn clean && yarn && yarn production-build` will give you any updated yarn.lock or jsii files.

Reply:
Thanks. I did `yarn && yarn build && yarn package` in [e59e763](https://github.com/aws-amplify/amplify-category-api/pull/2845/commits/e59e763d416cc2bbbd2d3e14dd79d98a9b85fb08). I think that did the trick, but I'm trying those commands now to see if there's a diff.

Reply:
No diff 😄